python/pytest.ini

```diff
@@ -1,5 +1,4 @@
 [pytest]
 markers =
     smoke_test: mark a test as a build verification testing.
-# TODO: Remove pubsub exclusion after the flakey tests are fixed
```

Comment:
Change this comment to remove this once pubsub tests job is merged to the main job